src/main/scala/com/evolutiongaming/scache/Releasable.scala

```diff
@@ -29,7 +29,7 @@ object Releasable {
   }
 
 
-  implicit def functorReleasable[F[_] : Applicative]: Functor[Releasable[F, *]] = new Functor[Releasable[F, *]] {
+  implicit def functorReleasable[F[_]]: Functor[Releasable[F, _]] = new Functor[Releasable[F, _]] {
```

Comment:
backward incompatible change.

You can do this differently: 

1. create new fixed version `implicit def functorReleasable`[F[_]]`
2. deprecate old one and remove `implicit`

Reply:
good catch, compiler was complaining about unused parameter and I didn't though that this is the part of public API.